draft-ietf-dnsop-svcb-https.md

```diff
@@ -1560,34 +1557,34 @@ In this document, this algorithm is referred to as "character-string decoding".
 The algorithm is the same as used by `<character-string>` in RFC 1035,
 although the output length in this document is not limited to 255 octets.
 
-## Decoding a value-list {#value-list}
+## Decoding a comma-separated list {#value-list}
 
-In order to represent lists of values in zone files, this specification uses
-an extended version of character-string decoding that adds the use of ","
-as a delimiter after double-quote processing.  When "," is not escaped
-(by a preceding "\\" or as the escape sequence "\\044"), it separates
-values in the output, which is a list of 1*OCTET.  (For simplicity, empty
-values are not allowed.)  We refer to this modified procedure as "value-list
-decoding".
+In order to represent lists of items in zone files, this specification uses
+comma-separated lists.  When "," is not escaped (by a preceding "\\"), it
+separates items in the list.  (For simplicity, empty items are not allowed.)
 
-    value-list = char-string
-    list-value = 1*OCTET
+    item            = 1*OCTET
+    ; item-allowed is OCTET minus "," and "\".
```

Comment:
What about double-quote?  (""")

Reply:
Double-quote is not special-cased here.  It is treated just like any other character, e.g, `alpn="\"foo\",bar"` -> `"foo",bar` -> (`"foo"`, `bar`).

Reply:
I should have re-read with more context.  Perhaps being clear below would help?